src/server/game/Entities/Player/Player.h

```diff
@@ -2574,6 +2574,8 @@ class TC_GAME_API Player : public Unit, public GridObject<Player>
         /***                    QUEST SYSTEM                   ***/
         /*********************************************************/
 
+        void _PushQuests();
```

Comment:
remove `_` from function name

Reply:
sure. I noticed other quests that are starting with _, I thought it's a notion for private methods.
When should one use _ for method names then?

Reply:
Usually it is reserved for class member variables, but looking at the start of this file, I presume there is an exception of some description for the utterly few methods starting with underscore.